packages/testsuite/cypress/e2e/elytron-oidc-client/test-oidc-security.cy.ts

```diff
@@ -0,0 +1,90 @@
+describe("TESTS: Configuration => Subsystems => Elytron OIDC client", () => {
+  let wildfly: string;
+  let ssoServer: string;
+
+  before(() => {
+    cy.startWildflyContainerSecured("standalone.xml")
+      .then((result) => {
+        wildfly = result as string;
+        cy.log("wildfly server endpoint: " + wildfly);
+      })
+      .then(() => {
+        cy.startSsoContainer("sso_server").then((result) => {
+          ssoServer = result as string;
+          cy.log("sso server endpoint: " + ssoServer);
+
```

Comment:
A comment that the following commands will secure the web console would be useful here.

No need to separate the commands by empty lines.